src/openforms/forms/api/viewsets.py

```diff
@@ -175,6 +174,11 @@ def configuration(self, request: Request, *args, **kwargs):
     retrieve=extend_schema(
         summary=_("Retrieve form details"),
         parameters=[UUID_OR_SLUG_PARAMETER],
+        description=_(
+            "Retrieve the details/configuration of a particular form. \n\nFor forms "
+            "that don't have translations enabled, the default language is activated, "
+            "otherwise the browser preferences determine the active language."
```

Comment:
The API response is about a single Form. So the language doesn't have to be generalised. More like:

> Iff the form has no translations enabled, its default language is forced with a `Set-Cookie` header and reflected in the `Content-Language` header. Normal HTTP Content Negotiation rules are applied/followed/honoured.

I don't envy the person that has to translate "iff", but I like it when documentation tells me the precise mechanisms and what to expect. I expect a reader of an OpenAPI spec to know or willing to lookup details of the HTTP specs.

Reply:
Your expectation is very optimistic, from personal experience, but I agree ;)